doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -620,13 +617,13 @@ file is a particular case of module called *library file*.
       all the recursive dependencies that were marked or transitively marked
       as ``Export``.
 
-   .. cmdv:: From @dirpath Require @qualid
-      :name: From ... Require ...
+.. cmd:: From @qualid Require {? {| Import | Export } } {+ @qualid }
+   :name: From ... Require ...
 
-      This command acts as :cmd:`Require`, but picks
-      any library whose absolute name is of the form :n:`@dirpath.@dirpath’.@qualid`
-      for some :n:`@dirpath’`. This is useful to ensure that the :token:`qualid` library
-      comes from a given package by making explicit its absolute root.
+   Works like :cmd:`Require`, but picks
+   any library whose absolute name is of the form :n:`@dirpath.@dirpath’.@qualid`
+   for some :n:`@dirpath’`. This is useful to ensure that the :token:`qualid` library
+   comes from a given package by making explicit its absolute root.
 
    .. exn:: Cannot load qualid: no physical path bound to dirpath.
```

Comment:
Some error message below contain quoting (`:n:`) which should be removed. Some, like this one, should gain some `@`:

```suggestion
   .. exn:: Cannot load @qualid: no physical path bound to @dirpath.
```